Heimdall/OAuthAuthorizationGrant.swift

```diff
@@ -14,12 +14,19 @@ public enum OAuthAuthorizationGrant {
     /// :param: refreshToken The refresh token.
     case RefreshToken(String)
 
+    /// An extension grant
+    ///
+    /// :param: grantType the grant type URI of the extension grant
+    /// :param: a dictionary of parameters
```

Comment:
Please add a parameter name for `[String: String]` to the documentation and use proper capitalization.